docs/CONFIG-VARS.md

```diff
@@ -72,14 +74,35 @@ NOTE: When deploying infrastructure into a private network (e.g. a VPN), with no
 
 NOTE: The script will either create a new Security Group, or use an existing Security Group, if specified in the `security_group_id` variable.
 
+### Public Access CIDRs
+
 You can use `default_public_access_cidrs` to set a default range for all created resources. To set different ranges for other resources, define the appropriate variable. Use an empty list [] to disallow access explicitly.
```

Comment:
default_public_access_cidrs works a bit differently than default_private_access_cidrs.  It's incorrect to say "Use an empty list [] to disallow access explicitly." in the public context, because the variable cluster_endpoint_public_access_cidrs works opposite to the others.  For cluster_endpoint_public_access_cidrs, when it's empty, it's wide open.  For this reason, the concept of a default variable here is problematic.  If you put something there to satisfy vm_public_access_cidrs, for example, then it must also be the full whitelist you need for cluster_endpoint_public_access_cidrs or you've locked out some sources.

I'd like to see this default variable dropped or at least changed to not apply to the cluster endpoint.  In lieu of that, we need the doc corrected.

Here are comments I put in the files a long time ago, which may be helpful in the doc here:

# Adding entries creates a restricting whitelist: set this to RESTRICT access on public endpoints to specified IPs.  Note that an empty array [] here allows ALL IPs access to the endpoint.  Customers using public endpoints will want this highly restricted.
#cluster_endpoint_public_access_cidrs    = ["149.173.0.0/16"] 

# For the 3 variables below you MUST add CIDRs to open up access for these services at their PUBLIC IPs.
# An empty array [] effectively disallows access because requisite security group ingress rules will not be created.
#vm_public_access_cidrs                  = ["149.173.0.0/16"] # Enable firewall rules for Public Jump/NFS VMs.
#postgres_public_access_cidrs            = ["149.173.0.0/16"] # Enable firewall rules for external postGres
#acr_public_access_cidrs

Reply:
Apologies for all the bold text, it's interpretting _some_ of my code comment character (#) as bold I think, and I'm not sure how to fix this.

Reply:
Hi @canpmh, please reach out this morning when you have a minute to discuss.

Reply:
Phil and I met to resolve the comments above. No doc changes were made based on current possible values for `postgres_public_access_cidrs`.